### PR TITLE
Be more explicit about integer width

### PR DESCRIPTION
**Pull request by fleupold**
Using int instead of uint256 can be confusing (especially since we use different width generously in our smart contracts).

E.g. #277 noticed that we are downcasting a uint256 to u128 although the value it gets assigned to is of type uint.

This PR changes all uses of uint to uint256 and also removes the unnecessary downcast.

### Test Plan

CI

### CODE REVIEW

File: contracts/StablecoinConverter.sol [hunk 1]
```diff
@@ -13,29 +13,29 @@ import "./libraries/TokenConservation.sol";
...
+    using SafeCast for uint256;
     using SafeMath for uint128;
     using BytesLib for bytes32;
     using BytesLib for bytes;
     using TokenConservation for int[];
     using TokenConservation for uint16[];
     using IterableAppendOnlySet for IterableAppendOnlySet.Data;
 
-    uint constant private MAX_UINT128 = 2**128 - 1;
+    uint128 constant private MAX_UINT128 = 2**128 - 1;
```
Comment (nlordell) [inline contracts/StablecoinConverter.sol]: Is it just me or is this constant unused and can be removed?

**Review by nlordell — COMMENTED** [review]
I'm always all for being more explicit!

**Review by bh2smith — APPROVED** [review]
Thanks! Thanks amaing

File: contracts/StablecoinConverter.sol (hunk 1 above)
Comment (bh2smith) [reply contracts/StablecoinConverter.sol]: This might be unused and lets please remove it here!

**Review by josojo — APPROVED** [review]
Great!

**Review by nlordell — APPROVED** [review]
:1st_place_medal:

**Pull request merged**